src/figuresClasses.ts

```diff
@@ -1,19 +1,76 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape,
+  color: Color,
+  getArea(): number,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape: Shape = 'triangle';
 
-}
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Invalid side length');
+    }
+
+    if ((a + b) <= c || (a + c) <= b || (b + c) <= a) {
+      throw new Error('It\'s not a triangle');
+    }
+  }
 
-export class Circle {
+  getArea(): number {
+    const p = 0.5 * (this.a + this.b + this.c);
 
+    return +(Math.sqrt(p
+      * (p - this.a)
+      * (p - this.b)
+      * (p - this.c)))
+      .toFixed(2);
```

Comment:
`return Number((Math.sqrt(
      p
      * (p - this.a)
      * (p - this.b)
      * (p - this.c),
    ).toFixed(2)
    ));`
    
    In my opinion, it will be better to use Number instead of '+'